dev/bots/test.dart

```diff
@@ -840,23 +840,14 @@ Future<void> _runFlutterPluginsTests() async {
       workingDirectory: checkout.path,
     );
     await runCommand(
-      pub,
-      <String>[
-        'global',
-        'activate',
-        'flutter_plugin_tools',
-      ],
-      workingDirectory: checkout.path,
-    );
-    await runCommand(
-      pub,
+      './script/incremental_build.sh',
```

Comment:
Do we need a `pub get` first, or is that already being explicitly done by an earlier step?

Reply:
It seems that incremental_build.sh does a pub get here:

https://github.com/flutter/plugins/blob/master/script/common.sh#L54

Reply:
Oops, I should have known that